Projects/EditCoInsured/Sources/Journeys/EditCoInsuredNavigation.swift

```diff
@@ -0,0 +1,263 @@
+import EditCoInsuredShared
+import Presentation
+import SwiftUI
+import hCore
+import hCoreUI
+
+public class EditCoInsuredNavigationViewModel: ObservableObject {
+    public init() {}
+
+    @Published var editCoInsuredConfig: InsuredPeopleConfig?
+    @Published var coInsuredInputModel: CoInsuredInputModel?
+    @Published var selectCoInsured: SelectCoInsured?
+    @Published var showProgressScreenWithSuccess = false
+    @Published var showProgressScreenWithoutSuccess = false
+
+    @Published var isEditCoinsuredSelectPresented: InsuredPeopleConfig?
+
+    @Published var externalNavigationRedirect = NavigationPath()
+}
+
+public enum EditCoInsuredScreenType {
+    case missingAlert
+    case newInsurance
+    case none
+}
+
+public struct EditCoInsuredNavigation: View {
+    let configs: [InsuredPeopleConfig]
```

Comment:
Maybe to split navigations into multiple navigations as we discussed before?
Wdyt?

Reply:
@sladan-hedvig did a split but it doesn't seem to work properly